### PR TITLE
DAOS-9784 dfuse: Tune dfuse tests for performance.

### PR DESCRIPTION
**Pull request by ashleypittman**
Add a dfuse mdtest test to get baseline performance
Tune daos_build process counts for faster results
Add a daos_build test with data caching only for focussing on metadata perf.

Signed-off-by: Ashley Pittman <ashley.m.pittman@intel.com>

### CODE REVIEW

File: src/tests/ftest/performance/mdtest_easy.py [hunk 1]
```diff
@@ -34,6 +35,16 @@ def test_performance_mdtest_easy_dfs_ec_16p2g1(self):
         """
         self.run_performance_mdtest(namespace="/run/mdtest_dfs_ec_16p2g1/*")
 
+    def test_performance_mdtest_easy_dfuse_s1(self):
+        """Test Description: Run MDTest Easy, POSIX dfuse, S1.
+
+        :avocado: tags=all,manual
+        :avocado: tags=hw,large
+        :avocado: tags=performance,performance_mdtest,performance_mdtest_easy,performance_dfuse
+        :avocado: tags=performance_mdtest_easy_dfuse_s1,dfuse
```
Comment (daltonbohning) [inline src/tests/ftest/performance/mdtest_easy.py:44]: The tags are usually in order (per line) of frequency, stage, feature, unique tags. Could we please move dfuse to line 3? Either of these
```
        :avocado: tags=all,manual
        :avocado: tags=hw,large
        :avocado: tags=performance,performance_mdtest,performance_mdtest_easy,performance_dfuse,dfuse
        :avocado: tags=performance_mdtest_easy_dfuse_s1
```
```
        :avocado: tags=all,manual
        :avocado: tags=hw,large
        :avocado: tags=dfuse
        :avocado: tags=performance,performance_mdtest,performance_mdtest_easy,performance_dfuse
        :avocado: tags=performance_mdtest_easy_dfuse_s1
```

**Review by daltonbohning — APPROVED** [review]
LGTM, with one request if you need to push again

**Review by mchaarawi — COMMENTED** [review]
looks fine to me, but i will leave my +1 to someone more familiar with test base file changes like Phil

**Review by saurabhtandan — APPROVED** [review]
Test changes LGTM

**Pull request merged**